README.md

```diff
@@ -45,11 +45,18 @@ or, if you haven't added your ssh key to your github account:
 git clone --recursive  https://github.com/Tribler/tribler.git
 ```
 
+Second install the [dependencies](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Linux.md)
+
+Third run setup.py
+```bash
```

Comment:
You don't need to install Tribler if you are running it from git, you can just execute the shell script as written in the next step.